tests/foreman/api/test_reporttemplates.py

```diff
@@ -947,11 +947,14 @@ def test_positive_installed_products(
         .read()
         .generate(data=input_data)
     )
+    assigned_lce = report[0]['Content View Environments'].split('/')[0]
+    assigned_cv = report[0]['Content View Environments'].split('/')[1]
+
     assert report, 'No report generated.'
     assert report[0]['Host Name'] == rhel_contenthost.hostname, 'Incorrect host was reported.'
     assert report[0]['Organization'] == org.name, 'Incorrect org was reported.'
-    assert report[0]['Lifecycle Environment'] == lce_name, 'Incorrect LCE was reported.'
-    assert report[0]['Content View'] == cv_name, 'Incorrect CV was reported.'
+    assert assigned_lce == lce_name, 'Incorrect LCE was reported.'
+    assert assigned_cv == cv_name, 'Incorrect CV was reported.'
```

Comment:
LCE and CV are no longer separate, so while this will work fine I would prefer it to be asserted as a single unit, with "Incorrect content view environment(s) reported." The same concept probably applies elsewhere in this PR as well.

Reply:
All right good point, I will rework that and apply this logic and assert those together then

Reply:
@jeremylenz Fixed